Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -1,26 +1,35 @@
-﻿namespace BootCamp.Chapter
+﻿using BootCamp.Chapter.Items;
+using System.Collections.Generic;
+
+namespace BootCamp.Chapter
 {
     public class Inventory
     {
-        private Item[] _items;
-        public Item[] GetItems()
-        {
-            return new Item[0];
-        }
+        private List<Item> _items;
+        public List<Item> Items { get { return new List<Item>(_items); } private set { _items = value; } }
```

Comment:
almost. you can remove the private `_items` and just use auto-implemented properties `{ get; set; }` on the public one.

also, note that since _items is a `List<Item>` if you were to keep this format, you wouldn't need `return new List<Item>(_items);` just `return _items;` but you wont need to keep it, more of an FYI :)

Reply:
Wouldn't it be passing by reference then? That way something else can change the internal items list

Reply:
That's a good point. The solution to this is to work with a readonly list. So private have List<Item>, publicly return ReadonlyList. It;s inneficient to create a list every time you need to return